backend/tests/test_integration_org_unit.py

```diff
@@ -2832,3 +2832,68 @@ def test_tree(self):
         for path, expected in util.get_fixture('test_trees.json').items():
             with self.subTest(path):
                 self.assertRequestResponse(path, expected)
+
+    def _create_conf_data(self):
+        import sqlite3
+        import mora.settings as settings
+        import os
+        os.remove(settings.USER_SETTINGS_DB_FILE)
+
+        defaults = {'show_roles': 'True',
+                    'show_user_key': 'False',
+                    'show_location': 'True'}
+
+        with sqlite3.connect(settings.USER_SETTINGS_DB_FILE,
+                             check_same_thread=False) as conn:
+            cur = conn.cursor()
+
+            create_query = ('CREATE TABLE orgunit_settings ' +
```

Comment:
Use a multiline string literal instead? (Also, this variable doesn't really add anything, does it? Inline it into its single use.)

Alternatively, just leave out the `+` — Python does the same as C when faced with consecutive string literals:
```python-console
>>> 'asd' 'asd'
'asdasd'
```